dcspy/starter.py

```diff
@@ -30,7 +30,8 @@ def _handle_connection(manager: KeyboardManager, parser: ProtocolParser, sock: s
     """
     start_time = time()
     LOG.info('Waiting for DCS connection...')
-    support_banner = _supporters(text='Huge thanks to: Alexander Leschanz, Sireyn, Nick Thain, BrotherBloat and others! For support and help! ', width=26)
+    support_banner = _supporters(text='Huge thanks to: Simon Leigh, Alexander Leschanz, Sireyn, Nick Thain, BrotherBloat and others! For support and help! ',
```

Comment:
@emcek , thanks sooo much for this. Much appreciated. ❤️

Reply:
Can't wait to see this scrolling across the LCD screen. 😄

Reply:
You really deserve it! Only for improving in G-Keys SDK, not mention rest of contributions.